.github/workflows/sync-config-schema.yaml

```diff
@@ -135,8 +135,8 @@ jobs:
           GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
           RELEASE_TAG: ${{ steps.release.outputs.release_tag }}
         run: |
+          rm -rf vcluster-config/ || true
```

Comment:
Was this on purpose ?

Reply:
yes, the CI failed because in some cases `vcluster-config` is already cloned in the previous step. We want to make sure that we commit to vcluster-config only changes made in this step (that's why we remove and clone again)